htslib/hts.h

```diff
@@ -1446,10 +1448,26 @@ static inline int hts_reg2bin(hts_pos_t beg, hts_pos_t end, int min_shift, int n
     return 0;
 }
 
+/// Compute the level of a bin in a binning index
+static inline int hts_bin_level(int bin) {
+    int l, b;
+    for (l = 0, b = bin; b; ++l, b = hts_bin_parent(b));
+    return l;
+}
+
+/// Compute the corresponding entry into the linear index of a given bin from
+/// a binning index
```

Comment:
This style of brief description has to be on a single `///` line. (See [doxygen docs](https://www.doxygen.nl/manual/docblocks.html#cppblock), “for the brief description”, third option.)